onnxruntime/core/providers/cpu/nn/qlinearconv.cc

```diff
@@ -561,6 +569,24 @@ Status QLinearConv<int8_t>::Compute(OpKernelContext* context) const {
                     static_cast<size_t>(group_input_channels),
                     static_cast<size_t>(input_image_size));
 
+      if (kernel_rank > 2) {
```

Comment:
for now, this should be kernel_rank != 2

Reply:
Fixed.

Reply:
Fixed